library/ecp_invasive.h

```diff
@@ -40,8 +40,33 @@ typedef enum {
     MBEDTLS_ECP_MOD_SCALAR
 } mbedtls_ecp_modulus_type;
 
+/* Requred macros for ECP split.
+ * If MBEDTLS_ECP_WITH_MPI_UINT is defined the new bignum interface is used.
+ */
+#ifndef MBEDTLS_ECP_WITH_MPI_UINT
```

Comment:
There seems to be a preference in existing code to use `#if !defined(FOO)` even when `#ifndef FOO` can be used